app/api/api_v1/routers/auth.py

```diff
@@ -18,13 +19,15 @@ async def login(db=Depends(get_db), form_data: OAuth2PasswordRequestForm = Depen
         extra={"props": {"user_id": form_data.username}},
     )
 
-    user = authenticate_user(db, form_data.username, form_data.password)
-    if not user:
+    authenticated = authenticate_user(db, form_data.username, form_data.password)
```

Comment:
Could just deconstruct the response to user, app_user here?

Reply:
authenticated could be `None` here